.buildkite/shared-pipeline-vars

```diff
@@ -5,7 +5,7 @@
 
 # The ~> modifier is not currently used, but we check for it just in case
 XCODE_VERSION=$(sed -E -n 's/^(~> )?(.*)/xcode-\2/p' .xcode-version)
-CI_TOOLKIT_VERSION='3.4.2'
+CI_TOOLKIT_VERSION='3.10.0'
```

Comment:
The DRYed `prepare_windows_host_for_node.ps1` command is available starting from this version.